STMicroelectronics/SPWF04Sx/SPWF04SxInterface.cs

```diff
@@ -792,14 +784,20 @@ public class SPWF04SxInterface : NetworkInterface, ISocketProvider, ISslStreamPr
             addresses = new[] { new IPEndPoint(IPAddress.Parse(result[1]), 80).Serialize() };
         }
 
-        public override string Id => nameof(SPWF04Sx);
-        public override string Name => this.Id;
-        public override string Description => string.Empty;
-        public override OperationalStatus OperationalStatus => this.State == SPWF04SxWiFiState.ReadyToTransmit ? OperationalStatus.Up : OperationalStatus.Down;
-        public override bool IsReceiveOnly => false;
-        public override bool SupportsMulticast => false;
-        public override NetworkInterfaceType NetworkInterfaceType => NetworkInterfaceType.Wireless80211;
+        public void Enable() => throw new NotImplementedException();
```

Comment:
So rename TurnOn and TurnOff above to Enable and Disable and remove them from here